src/make-table-row.js

```diff
@@ -0,0 +1,40 @@
+// //ok so this function creates a ROW
+// //that looks like this:
+
```

Comment:
If you are not using the code, delete it from your repo. It adds clutter and can make things more confusing in a production setting.

Reply:
Awesome catch, thank you so much! Fixed! :)